frontend/.prettierrc

```diff
@@ -1 +1,4 @@
-{}
+{
+  "semi": false,
+  "singleQuote": true
```

Comment:
See my latest comment here - please use the config found here for quotes: https://stackoverflow.com/questions/55430906/prettier-single-quote-for-javascript-and-json-double-quote-for-html-sass-and-c